modules/hcp-dc-mgmt-cluster.adoc

```diff
@@ -6,142 +6,23 @@
 [id="hcp-dc-mgmt-cluster_{context}"]
 = Setting up a management cluster for {hcp} in a disconnected environment
 
-To set up an {product-title} management cluster, you can use dev-scripts, or if you are based on virtual machines, you can use the `kcli` tool. The following instructions are specific to the `kcli` tool.
+To set up an {product-title} management cluster, you can use dev-scripts.
```

Comment:
dev-scripts is still for a development environment. I'm not sure we should suggest that to customers.

Reply:
Removed. Thanks!